apps/meteor/server/services/team/service.ts

```diff
@@ -633,14 +633,19 @@ export class TeamService extends ServiceClassInternal implements ITeamService {
 			};
 		}
 
-		const users = await Users.findActive({ ...query }).toArray();
-		const userIds = users.map((m) => m._id);
-		const cursor = TeamMember.findMembersInfoByTeamId(teamId, count, offset, {
-			userId: { $in: userIds },
-		});
+		const cursor = this.TeamMembersModel.findMembersInfoByTeamId(teamId, count, offset);
+		const records = await cursor.toArray();
+		const teamUserIds = records.map((m) => m.userId);
```

Comment:
```suggestion
		const teamMembers = await this.TeamMembersModel.findMembersInfoByTeamId(teamId, count, offset).toArray();
		const teamUserIds = teamMembers.map((m) => m.userId);
```
Maybe we could change the const names here to improve legibility